addons/mail/static/src/models/popover_view.js

```diff
@@ -252,6 +261,13 @@ registerModel({
             },
             inverse: 'popoverViews',
         }),
+        messageContextMenuView: one('MessageContextMenu', {
+            compute() {
+                    return {};
+            },
+            inverse: 'messageContextPopoverView',
+        }),
```

Comment:
`messageContextMenuView` is a kind of content for the popover view.
Here current code makes it so that there's always a `MessageContextMenuView`.
This is not right: `PopoverView` assumes there's only 1 content at once.

So you should conditionally `return {}`.
I see no field that defines the "owner" in the context of `MessageView/onContextMenu`.

Likely need to define this:
```js
messageViewOwnerAsContextMenu: one('MessageView', {
   identifying: true,
   inverse: 'messageContextPopoverView',
}),
```

And conditionally have `messageContextMenuView` based on `messageViewOwnerAsContextMenu` being set.